client/components/disputed-order-notice/index.js

```diff
@@ -124,67 +208,28 @@ const DisputeNeedsResponseNotice = ( {
 		} );
 	}, [ isPreDisputeInquiry, disputeReason, countdownDays ] );
 
-	const titleStrings = {
-		// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
-		dispute_default: __(
-			// eslint-disable-next-line max-len
-			'This order has been disputed in the amount of %1$s. The customer provided the following reason: %2$s. Please respond to this dispute before %3$s.',
-			'woocommerce-payments'
-		),
-		// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
-		inquiry_default: __(
-			// eslint-disable-next-line max-len
-			'The card network involved in this order has opened an inquiry into the transaction with the following reason: %2$s. Please respond to this inquiry before %3$s, just like you would for a formal dispute.',
-			'woocommerce-payments'
-		),
-		// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
-		dispute_urgent: __(
-			'Please resolve the dispute on this order for %1$s labeled "%2$s" by %3$s.',
-			'woocommerce-payments'
-		),
-		// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
-		inquiry_urgent: __(
-			'Please resolve the inquiry on this order for %1$s labeled "%2$s" by %3$s.',
-			'woocommerce-payments'
-		),
-	};
-
-	let buttonLabel = __( 'Respond now', 'woocommerce-payments' );
-	let suffix = '';
-
-	let titleText = isPreDisputeInquiry
-		? titleStrings.inquiry_default
-		: titleStrings.dispute_default;
-
-	// If the dispute is due within 7 days, adjust wording and highlight urgency.
-	if ( countdownDays < 7 ) {
-		titleText = isPreDisputeInquiry
-			? titleStrings.inquiry_urgent
-			: titleStrings.dispute_urgent;
-
-		suffix = sprintf(
-			// Translators: %s is the number of days left to respond to the dispute.
-			_n(
-				'(%s day left)',
-				'(%s days left)',
-				countdownDays,
-				'woocommerce-payments'
-			),
-			countdownDays
-		);
-	}
+	const isUrgent = countdownDays < 7;
 
-	const title = sprintf(
-		titleText,
-		formattedAmount,
-		reasons[ disputeReason ].display,
-		dateI18n( 'M j, Y', dueBy.local().toISOString() )
-	);
+	const buttonLabel = isUrgent
+		? __( 'Respond today', 'woocommerce-payments' )
```

Comment:
The behaviour of buttonLabel changes from base branch. It says `Respond today` when countdownDays < 1 but now when countdownDays < 7.

Is that intentional?

UX-wise IMO is fine. 'Respond today' and 'Respond now' to me both urge some urgency.

Reply:
Good catch, I'll fix this so it's consistent with previous behaviour. Perils of refactoring :)

Reply:
Fixed in 3e7fa9e109f0e20c469566a4a55aae0e88375dce – I want to ensure the button is consistent with existing UI, no good reason to change it.